### PR TITLE
[Improvement] vpc/vpc-nat-gateway - Monitor bandwidth

### PR DESCRIPTION
**Pull request by michaelwittig**

### CODE REVIEW

File: vpc/vpc-nat-gateway.yaml [hunk 1]
```diff
@@ -95,6 +95,74 @@ Resources:
       Dimensions:
       - Name: NatGatewayId
         Value: !Ref NatGateway
+  AlarmNatGatewayBahndwith:
```
Comment (andreaswittig) [inline vpc/vpc-nat-gateway.yaml]: `AlarmNatGatewayBandwith`

File: vpc/vpc-nat-gateway.yaml [hunk 2]
```diff
@@ -95,6 +95,74 @@ Resources:
...
+            Namespace: 'AWS/NATGateway'
+            MetricName: BytesOutToSource # bytes per minute
+            Dimensions:
+            - Name: NatGatewayId
+              Value: !Ref NatGateway
+          Period: 600
+          Stat: Average
+          Unit: Bytes
+        ReturnData: false
+      - Expression: '(in1+in2+out1+out2)/60*8/1000/1000/1000' # to Gbit/s
```
Comment (andreaswittig) [inline vpc/vpc-nat-gateway.yaml]: The expression counts traffic twice, doesn't it?

Comment (michaelwittig) [reply vpc/vpc-nat-gateway.yaml]: I wasn't sure about this as well. https://aws.amazon.com/premiumsupport/knowledge-center/cloudwatch-nat-gateway-bandwidth/ states that we have to add all 4 numbers.

I performed a test:
EC2 a in private subnet
EC2 b in public subnet

I was sending 0.5 Gibt/s from a to b and measured the exact same number with the formula. So I guess it's correct?

File: vpc/vpc-nat-gateway.yaml (hunk 1 above)
Comment (michaelwittig) [reply vpc/vpc-nat-gateway.yaml]: fixed

File: vpc/vpc-nat-gateway.yaml (hunk 2 above)
Comment (michaelwittig) [reply vpc/vpc-nat-gateway.yaml]: Let me create a support request.

Comment (michaelwittig) [reply vpc/vpc-nat-gateway.yaml]: ok. I have to divide by 2 :)

**Pull request merged**